### DISCUSSION TITLE
Why does the same code running under different mmcv and mmsegmentation versions result in higher results for the lower version

### DISCUSSION THREAD

**Discussion by Lucaslifeday** [open-mmlab/mmsegmentation#2704] [General]
For example, I reached 90 miou with mmcv 1.2.7 and mmsegmentation 0.13.0.
With mmcv 1.6.0 and mmsegmentation 0.28.0 the miou reaches 89?
The lower version will work better instead

**Comment by Lucaslifeday** [comment]
Same dataset, data enhancement, number of training rounds, etc.

**Comment by erdikilic** [comment]
Did you use same seed?

**Comment by Lucaslifeday** [comment]
Thank you for your answer. I fixed seed. I feel that the problem is caused by the weight of pre-training. Use of pre-training weights that do not correspond to the mmsegmentation version will have an impact.
